README.md

```diff
@@ -42,15 +37,56 @@ under mycroft.conf
   },
 
   // Mechanism used to play WAV audio files
-  // Override: SYSTEM
   "play_wav_cmdline": "paplay %1 --stream-name=mycroft-voice",
 
   // Mechanism used to play MP3 audio files
-  // Override: SYSTEM
   "play_mp3_cmdline": "mpg123 %1",
 
   // Mechanism used to play OGG audio files
-  // Override: SYSTEM
   "play_ogg_cmdline": "ogg123 -q %1"
 }
-```
\ No newline at end of file
+```
+
+## Using Legacy AudioService
+
+The legacy audio service supports audio playback via the old mycroft api ([@mycroft](https://github.com/MycroftAI/mycroft-core/blob/dev/mycroft/skills/audioservice.py#L43) [@ovos](https://github.com/OpenVoiceOS/ovos-bus-client/blob/dev/ovos_bus_client/apis/ocp.py#L51))
+
+by default OCP acts as a translation layer for this api and no action is needed, but if you want to disable ocp this api remains available
+
+> **NOTE:** once ovos-media is released OCP and this api will be disabled by default and deprecated!
+
+```javascript
```

Comment:
suggestion(non-blocking): use \`\`\`json for better formatting, potentially?

Reply:
markdown rendering in github doesnt like comments, thats why i use javascript usually, avoids the red highlights as it would be invalid json but is a valid javascript comment